dashboard/app/models/ability.rb

```diff
@@ -411,27 +411,21 @@ def initialize(user)
       end
     end
 
-    # Checks if user is directly enrolled in pilot or has a teacher enrolled
     if user.persisted?
-      if user.permission?(UserPermission::LEVELBUILDER) ||
-        user.has_pilot_experiment?(CSA_PILOT) ||
-        user.teachers.any? {|t| t.has_pilot_experiment?(CSA_PILOT)} ||
-        user.has_pilot_experiment?(CSA_PILOT_FACILITATORS) ||
-        user.teachers.any? {|t| t.has_pilot_experiment?(CSA_PILOT_FACILITATORS)}
-
-        can :get_access_token, :javabuilder_session
+      # Checks if user is a verified instructor or the student of a verified instructor.
```

Comment:
minor, but could we update the wording of these comments to be more consistent so that it's clear what permission each route grants to whom? maybe something along the lines of
`# All verified instructors and students of verified instructors can run code on Java Lab levels`
`# Only verified instructors can access and run Java Lab exemplars`
`# Only levelbuilders can access and update Java Lab validation code`